text_mining.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+"""
+YOUR HEADER COMMENT HERE
+
+@author: ISA BLANCETT
+
+"""
+
+# Assumptions:
+# Wikipedia articles of states are up to date
+# Distance between states = google maps distance between states
+# If states are next to each other, their distance is 0
+# Hawaii is considered 5000 miles from Alabama for the sake of this script
+
+import wikipedia
+from collections import Counter
+from itertools import chain
```

Comment:
I don't see where you are using this chain. If you are not using this library anymore, please delete it from your final code!